File: samples/Java/arbitrary-precision-integers--included-.java
import java.math.BigInteger;

class Program {
    public static void main(String[] args) {
        BigInteger x = BigInteger.valueOf(5).pow(BigInteger.valueOf(4).pow(BigInteger.valueOf(3).pow(2).intValue()).intValue());
        String y = x.toString();
        int l = y.length();
        System.out.printf("5**4**3**2 = %s...%s and has %d digits\n",
                          y.substring(0,20), y.substring(l-20), l);
    }
}

File: samples/Dart/arbitrary-precision-integers--included-.dart
void main() {
  var s = pow(5, pow(4, pow(3, 2))).toString();

  print('contains given digits: ${s.startsWith('62060698786608744707') && s.endsWith('92256259918212890625')}');
  print('number of digits: ${s.length}');
}
